pilot/pkg/config/file/store.go

```diff
@@ -530,6 +530,16 @@ func TranslateObject(obj *unstructured.Unstructured, domainSuffix string, schema
 		panic(err)
 	}
 
+	// attempt to handle status if we know the type
+	statusStruct, err := schema.Status()
```

Comment:
is this going to make `statusStruct` non-nil if we don't find a `status` field?

Reply:
I would have thought if so and it matter some test would fail, so maybe not a big deal but I think it may be safer to only set this if the status field is present.

Reply:
should be addressed in [830ef08](https://github.com/istio/istio/pull/55055/commits/830ef08ce49c054ab3a2468fd7e86cea6a168a8f)